roles/teuthology_nameserver/tasks/dnssec-keygen.yml

```diff
@@ -0,0 +1,30 @@
+#Moved dnssec key generation from deploy-teuthology script to the ansible task due to HMAC-MD5 being deprecated on Tumbleweed.
+- name: Generate new keys
+  shell: "/usr/sbin/dnssec-keygen -r /dev/urandom -a HMAC-MD5 -b  512 -n HOST '{{ zone_name }}'"
+  register:
+    dns_keygen
+
+- name: Get prikey contents
+  shell: cat {{ dns_keygen.stdout }}.private
+  register: prikey_out
+
+- name: Get prikey secret
+  set_fact: 
+    secret: "{{ (prikey_out.stdout | from_yaml).Key }}"
+
+- name: Set key vars
+  set_fact:
+    dns_key: "{{ dns_keygen.stdout }}"
+    pubkey_name: "{{ dns_keygen.stdout }}.key"
+    prikey_name: "{{ dns_keygen.stdout }}.private"
+    zone_secret: "{{ secret }}"
+
+- name: Fetch the keys to secrets local path
+  fetch:
+    src: /home/{{ ansible_ssh_user }}/{{ item }}
```

Comment:
`/home/{{ ansible_ssh_user}}` prefix is extra.

Reply:
```TASK [teuthology_nameserver : Fetch the keys to secrets local path] **************************************
task path: /home/kyr/suse/teuthology-deploy/roles/teuthology_nameserver/tasks/dnssec-keygen.yml:22
failed: [51.91.148.55] (item=Ksbg.suse.de.+157+10241.key) => changed=false 
  ansible_loop_var: item
  invocation:
    module_args:
      src: /home/root/Ksbg.suse.de.+157+10241.key
  item: Ksbg.suse.de.+157+10241.key
  msg: 'file not found: /home/root/Ksbg.suse.de.+157+10241.key'
```

Reply:
Indeed my bad cause I was using `opensuse` user so didn't notice. I changed it to `ansible_env.HOME` and it works as it should